tests/testthat/_snaps/eval-select.md

```diff
@@ -4,8 +4,7 @@
       (expect_error(select_loc(mtcars, f())))
     Output
       <error/rlang_error>
-      Error in `select_loc()`:
-      Caused by error in `f()`:
+      Error in `f()`:
```

Comment:
Can we keep the chained error here? Chained error support is one the main feature of this release so I'd like to keep them.